lib/PuppeteerSharp/Constants.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace PuppeteerSharp
+{
+    /// <summary>
+    /// Constants common to devtools API interaction
+    /// </summary>
+    internal class Constants
+    {
+        public const string ID = "id";
```

Comment:
Looks like you found a dynamic I missed.

If I've understood you correctly, you're suggesting a large amount of new classes and changes to avoid all places where a lookup on a JToken happens, which I see a beyond the scope of simply tidying up dynamics.
This isn't something I have time for, sorry.

Reply:
No worries @tommymonk